src/components/Settings/Profile/NFTPicture.tsx

```diff
@@ -1,18 +1,15 @@
 import { LensHubProxy } from '@abis/LensHubProxy';
-import { useLazyQuery, useMutation } from '@apollo/client';
 import IndexStatus from '@components/Shared/IndexStatus';
 import { Button } from '@components/UI/Button';
 import { ErrorMessage } from '@components/UI/ErrorMessage';
 import { Form, useZodForm } from '@components/UI/Form';
 import { Input } from '@components/UI/Input';
 import { Spinner } from '@components/UI/Spinner';
 import useBroadcast from '@components/utils/hooks/useBroadcast';
-import type { Mutation, NftImage, Profile, UpdateProfileImageRequest } from '@generated/types';
-import {
-  CreateSetProfileImageUriTypedDataDocument,
-  CreateSetProfileImageUriViaDispatcherDocument,
-  NftChallengeDocument
-} from '@generated/types';
+import type { NftImage, Profile, UpdateProfileImageRequest } from '@generated/types';
+import { useCreateSetProfileImageUriViaDispatcherMutation } from '@generated/types';
+import { useCreateSetProfileImageUriTypedDataMutation } from '@generated/types';
+import { useNftChallengeLazyQuery } from '@generated/types';
```

Comment:
Why are these not combined as well? 👀

Reply:
fixed in https://github.com/lensterxyz/lenster/pull/1194/commits/200bc00f75fd6013c9616d353a1fa75940715478

not sure, need to find eslint rule for this 😕